rfd/0026/README.md

```diff
@@ -2495,3 +2503,31 @@ communicate it in some way (e.g in the API documentation or by making changes to
 the API/command line tools so that the impact on shared volumes is more
 obvious).
 
+### Automatic mounting for non-docker containers
```

Comment:
Instead of adding this content to the "Open questions" section, maybe we should create a "Platform changes" section if it doesn't already exist and put this content there. In a separate change we would also document in the same section the changes that were already made to `dockerinit`.